Linguini.Syntax/Parser/LinguiniParser.cs

```diff
@@ -602,13 +598,22 @@ private bool TryGetTextSlice([NotNullWhen(true)] out TextSlice? textElement, out
             var startPos = _reader.Position;
             var textElementType = TextElementType.Blank;
 
-            while (_reader.TryPeekCharSpan(out var span))
+            int index;
+            while ((index = _reader.IndexOfAnyChar(" \n\r{}")) != -1)
```

Comment:
This needs to be a different function or to write is as such: 
```suggestion
            while ((index = _reader.IndexOfAnyChar(" \n\r{}".AsSpan())) != -1)
```